python/sdk/prelude_sdk/controllers/build_controller.py

```diff
@@ -102,6 +105,24 @@ def create_threat(self, name, published, threat_id=None, source_id=None, source=
             return res.json()
         raise Exception(res.text)
 
+    @verify_credentials
+    def create_threat_from_directory(self, directory: str, **kwargs):
+        tests = list()
+        uploads = list()
+        try:
+            for technique_dir in os.listdir(directory):
+                with open(f'{directory}/{technique_dir}/test.go', 'r') as f:
```

Comment:
why not creating the IOAs too? just try/ignore 403s

Reply:
the output of threat intel gen is a yaml. Could probably convert it to whatever we need, but it sounded like we were just ignoring these until we figured out the sigma stuff and then we would have the threat intel stuff output a sigma definition instead.

Reply:
ah. fair. kk. i'll update this later then when we figure out the IOA format